project/AppDependencies.scala

```diff
@@ -2,20 +2,20 @@ import sbt.*
 
 object AppDependencies {
 
-  private lazy val mongoHmrcVersion     = "1.7.0"
-  private lazy val bootstrapPlayVersion = "8.4.0"
+  private lazy val mongoHmrcVersion     = "2.0.0"
+  private lazy val bootstrapPlayVersion = "8.6.0"
 
   private lazy val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"         % mongoHmrcVersion,
     "uk.gov.hmrc"       %% "bootstrap-frontend-play-30" % bootstrapPlayVersion,
-    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "8.5.0"
+    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "10.1.0"
   )
 
   private lazy val test: Seq[ModuleID] = Seq(
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-test-play-30" % mongoHmrcVersion,
     "uk.gov.hmrc"       %% "bootstrap-test-play-30"  % bootstrapPlayVersion,
     "org.jsoup"          % "jsoup"                   % "1.17.2",
-    "org.mockito"       %% "mockito-scala-scalatest" % "1.17.30",
+    "org.mockito"       %% "mockito-scala-scalatest" % "1.17.31",
     "org.scalatestplus" %% "scalacheck-1-17"         % "3.2.18.0"
```

Comment:
Please bump this to `1-18` as per [here](https://github.com/scalatest/scalatestplus-scalacheck?tab=readme-ov-file#scalatest-plus-scalacheck), thanks